update docs for lily v0.10.0 (#196)
filecoin-project/sentinel: docs/content/en/software/lily/operation.md

## API endpoint

> When the daemon is started, the repo path is where this communication coordination is handled. The repo path contains information about where the JSON RPC API is (`api` file located in repo path) located, along with an authentication token used (`token` file) for writeable interactions. For more details, see the [Lotus API documentation](https://docs.filecoin.io/build/lotus/).

Reviewer: ```suggestion
When the daemon is started, the repo path is where this communication coordination is handled. The repo path contains information about where the JSON RPC API is (`api` file located in repo path) located, along with an authentication token used (`token` file) for writeable interactions. For more details, see the [Lotus API documentation](https://lotus.filecoin.io/reference/basics/api-access/).
```

Previous link is dead.

The IP/port to which the daemon binds can be managed via the `--api` flag. By default, the daemon binds to `localhost` and will be unavailable externally unless bound to a publicly-accessible IP.

---

## Jobs

> Once the Lily daemon process is [running and fully synced](setup.md), you may manage its Jobs through the CLI. (Note: behavior is undefined if Lily is not fully synced, so one may gate the Job creation to wait for the sync to complete with `lily sync wait && lily ...`.).

Reviewer 2: ```suggestion
Once the Lily daemon process is [running and fully synced](setup.md), you may manage its Jobs through the CLI.

{{< alert >}}
**Instructions are the equivalent for `lotus` and `lotus-miner`**. For simplicity, we will just show how to do it with `lotus` by editing the [Lotus Node configuration]({{< relref "defaults" >}}), but the same can be achieved by editing the [Lotus Miner configuration]({{< relref "configuration" >}}).
{{< /alert >}}
```

Jobs may be executed on a lily node via the `job run` command. The `job run` command accepts the following flags:

> - `--window` duration after which job execution will be canceled while processing a tipset.

Reviewer 2: Maybe it would be good to add an example with the acceptable units of time.

> - `--tasks` specify the list of [tasks](#tasks) to run by this job. Some tasks [will be heavier than others](hardware.md). A tipset is only processed when all tasks in the job have finished. Only then will Lily move to the next epoch. The default value of this flag if omitted is all tasks lily is capable of performing.

Reviewer 2: ```suggestion
- `--tasks` specify the list of [tasks](#tasks) to run. Some tasks [will be heavier than others](hardware.md). A tipset is only processed when all the specified tasks have finished. Only then will Lily move on to the next epoch. The default value of this flag is all tasks lily is capable of performing.
```

Removing some verbosity.

- `--storage` specifies which of the configured storage outputs the job will write to.
- `--name` specifies the name of the job for easy identification later. The provided value
  will appear as `reporter` in the `visor_processing_reports` table.

> - `--restart-delay` duration to wait before restarting job after it ends execution.

Reviewer 2: Also would be good to add an example with the acceptable units of time here.

- `--restart-on-failure` specifies if a job should be restarted if it fails.
- `--restart-on-completion` specifies if a job should be restarted once it completes.

> Currently, Lily is capable of executing the following types of jobs:

Reviewer 2: Seems like this is supposed to enumerate the following subsections with the header 3 formatting.

> Currently, Lily is capable of executing the following types of jobs:

Reviewer 2: ```suggestion
Currently, Lily is capable of executing the following types of jobs:

- [Watch](#watch)
- [Walk](#walk)
- [Find](#find)
- [Fill](#fill)
- [Survey](#survey)
```

It might also make more sense to move this sooner (before [L28?](https://github.com/filecoin-project/sentinel/pull/196/files#diff-217c446f189cba7ba31958b1223a8583d09dc6b3c16587480eb31c6cfe874a76R28)).

> Currently, Lily is capable of executing the following types of jobs:

Author: ![image](https://user-images.githubusercontent.com/6546409/172271675-51eba516-4087-4c86-a6eb-cf1d9bf7585c.png)

    PasswordEnv = "LILY_REDIS_PASSWORD"
    DB = 0
    PoolSize = 0
[Queue.Workers.Worker1.WorkerConfig]
    # Defines how many tasks a worker may process in parallel
    Concurrency = 1
    # Logging level of internal task system
    LoggerLevel = "debug"
    # Priorities of corresponding queues:
    #   processes Watch tasks 50% of the time.
    #   processes Fill tasks 30% of the time.
    #   processes Index tasks 10% of the time.
    #   processes Walk tasks 10% of the time
    WatchQueuePriority = 5
    FillQueuePriority = 3
    IndexQueuePriority = 1
    WalkQueuePriority = 1
    # If true, the queues with higher priority are always processed first.
    # Else queues with lower priority are processed only if all the other queues with higher priorities are empty.
    StrictPriority = false
    # Specifies the duration to wait to let workers finish their tasks
    ShutdownTimeout = 30000000000
```

## Tasks

Lily provides several tasks to capture different aspects of the blockchain state. The type of data extracted by Lily is controlled by the below tasks. Jobs accepts tasks to run as a comma-separated list. The data extracted by a task is stored in its related Models.

> | Name                              |                            Model                             | Duration |

Reviewer 3: Clarify column title.
```suggestion
| Name                              |                            Model                             | Processing Duration |
```
